.pre-commit-config.yaml

```
@@ -188,3 +188,8 @@ repos:
         entry: 'pg8000'
         files: ^ci/deps
         types: [yaml]
+    -   id: validate-min-versions-in-sync
+        name: Check minimum version of dependencies are aligned
+        entry: python scripts/validate_min_versions_in_sync.py
+        language: python
+        files: ^(ci/deps/actions-*-minimum_versions\.yaml|pandas/compat/_optional\.py)$
```

This is regex (not glob), I presume you meant
```suggestion
        files: ^(ci/deps/actions-.*-minimum_versions\.yaml|pandas/compat/_optional\.py)$
```
?